tests/test_lodcm.py

```diff
@@ -34,20 +152,20 @@ def test_lodcm_destination(fake_lodcm):
     for d in dest:
         assert isinstance(d, str)
 
-    lodcm.h1n.move('OUT')
+    lodcm.h1n_state.move('OUT')
     assert len(lodcm.destination) == 1
-    lodcm.h1n.move('C')
+    lodcm.h1n_state.move('C')
     assert len(lodcm.destination) == 2
     # Block the mono line
     lodcm.yag.move('IN')
     assert len(lodcm.destination) == 1
-    lodcm.h1n.move('Si')
+    lodcm.h1n_state.move('Si')
     assert len(lodcm.destination) == 0
     lodcm.yag.move('OUT')
     assert len(lodcm.destination) == 1
```

Comment:
I have added some extra lines that test the `lodcm.main_line` and `lodcm.mono_line` here: https://github.com/pcdshub/pcdsdevices/pull/766/commits/7556127dd33e120a638158b57380b6bea5e5606a

I left the `len` the way it is, because that was somebody else's tests, and also because I think it might be important to test to make sure only one line is in certain situation, or no line at all or both...